omod/src/main/java/org/openmrs/module/registrationapp/form/RegisterPatientFormBuilder.java

```diff
@@ -264,21 +264,25 @@ public static void resolvePatientIdentifierFields(NavigableFormStructure form, P
 							log.warn("Multiple values for a single patient identifier type not supported, ignoring extra values");
 						}
 						String parameterValue = parameterValues[0];
-						if (StringUtils.isNotBlank(parameterValue)) {
-							PatientIdentifierType identifierType = Context.getPatientService().getPatientIdentifierTypeByUuid(field.getUuid());
+						if (StringUtils.isNotBlank(parameterValue)||(StringUtils.isEmpty(parameterValue))) {
+							PatientIdentifierType identifierType = Context.getPatientService().getPatientIdentifierTypeByUuid(field.getUuid());					   
+			
 							if (identifierType  != null) {
 
 								// see if there is existing identifier with this value, if so, no need to update
 								for (PatientIdentifier oldIdentifier : patient.getPatientIdentifiers(identifierType)) {
-									if (oldIdentifier.getIdentifier().equals(parameterValue)) {
+									if (oldIdentifier.getIdentifier().equals(parameterValue)) {					
 										return;
+									} else if(oldIdentifier.getIdentifier().isEmpty()) {
+									   log.warn("you have an empty identifier fied");  
```

Comment:
@sherrif10 ,thanks for this,you can change fied to field around this line

Reply:
Thanks @HerbertYiga will resolve that